src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <div class="col-sm-12 col-lg-12">
+      <div>
+          <cd-table-actions class="btn-group mb-4"
+              [permission]="permission"
+              [selection]="selection"
+              [tableActions]="createTableActions">
+          </cd-table-actions>
+      </div>
```

Comment:
Thanks @epuertat , I am working on improving the UI as you suggested.

Reply:
![Screenshot from 2023-02-21 13-11-45](https://user-images.githubusercontent.com/66050535/220279703-242b397c-7955-4f01-b6ea-42bd79129f26.png)


@epuertat , Does it look better? Another thing am not sure is whether to show the Edit and delete buttons only on the clicked node..or we want to show it by default for every entity(which looks a bit redundant)

Reply:
![Screenshot from 2023-02-21 18-46-38](https://user-images.githubusercontent.com/66050535/220355078-4e6ef575-d304-4c72-8f8e-b96eec3263ca.png)

Reply:
Thanks! Yup, the former looks better.

Why not adding a "+" button and a "root" node (even if the name is hidden), so that we can get rid of the "Create" button?

I checked Patternfly and they suggest [making the actions visible on `hover`/`mouseover`](https://www.patternfly.org/v4/components/tree-view/design-guidelines#adding-action-items-to-a-tree-view) (instead of `onclick`).

Additionally I'd put the actions closer to the item: with this move we want to make it easier for users to create/edit/delete stuff and avoid extra mouse moves, so the closer the better (with some padding of course).

Regarding the "(zone)", "(realm)", etc. suffix, I'd add some visual decoration to make it look different from the item name: maybe using a label (as I suggested before), or turning it gray/italics, ....